x/feemarket/keeper/eip1559_test.go

```diff
@@ -98,12 +97,13 @@ func (suite *KeeperTestSuite) TestCalculateBaseFee() {
 			suite.app.FeeMarketKeeper.SetBlockGasWanted(suite.ctx, tc.parentBlockGasWanted)
 
 			// Set next block target/gasLimit through Consensus Param MaxGas
-			blockParams := abci.BlockParams{
-				MaxGas:   100,
-				MaxBytes: 10,
-			}
-			consParams := abci.ConsensusParams{Block: &blockParams}
-			suite.ctx = suite.ctx.WithConsensusParams(&consParams)
+			// Todo: Update the block gas through the consensus params keeper
+			// blockParams := abci.ConsensusParams{
+			// 	MaxGas:   100,
```

Comment:
@fedekunze maybe its time to address: https://github.com/evmos/ethermint/issues/1231 ?